### ISSUE TITLE
Support tidal information

### ISSUE THREAD

**Issue by sonora** [osmandapp/OsmAnd#16374]
I received a somewhat vague request to support factoring in tidal information (tide times, tide charts,  tidal corrections) in our Nautical maps plugin. We would need to research where to get what open source data and in what format so that it could be displayed for practical value add.

I just googled a bit and found some example data e.g. here https://www.tide-forecast.com (not necessarily open source).

**Comment by vshcherb** [comment]
Please check if there are layers on  GFS (https://www.ncei.noaa.gov/products/weather-climate-models/global-forecast) or ECWMF https://charts.ecmwf.int/

**Comment by sonora** [comment]
Yes, there is an abundance of data at these sources. What file/data format would we be preferably looking at?

**Comment by DmitryAlexei** [comment]
Same
- https://github.com/osmandapp/OsmAnd/issues/20796
